### PR TITLE
Task 156308466 - Supplementary Information tab

### PR DESCRIPTION
**Pull request by monicajianu**
* Implemented React component for the Supplementary Information tab
* Added Supplementary Information content generation in the ExperimentPage controller

**Note**: this branch is based on branch task_155186621, only commits of 3rd and 4th April are new.

### CODE REVIEW

File: sc/src/main/javascript/bundles/experiment-page/src/SupplementaryInformationPageRoute.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+const SectionContent = ({type, props}) => {
+  const Section = sectionTypeComponent[type]
+
+  return (
+    Section ? <Section {...props} /> : null
+  )
+
+}
+const SupplementaryInformationPageRoute = (props) => {
+  console.log(props)
```
Comment (alfonsomunozpomer) [inline sc/src/main/javascript/bundles/experiment-page/src/SupplementaryInformationPageRoute.js]: Remove! ;)

File: sc/src/main/javascript/bundles/experiment-page/src/supplementary-information/StaticTable.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      </tbody>
+    </table>
+  </div>
+)
+
+Table.propTypes = {
+  data: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.string)).isRequired
+}
+
+export default Table
```
Comment (alfonsomunozpomer) [inline sc/src/main/javascript/bundles/experiment-page/src/supplementary-information/StaticTable.js]: I feel this component’s name is a bit too generic... Maybe `AnalysisTable`?

File: sc/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentController.java [hunk 3]
```diff
@@ -1,29 +1,47 @@
 package uk.ac.ebi.atlas.experimentpage;
 
-import com.google.gson.Gson;
-import com.google.gson.JsonArray;
-import com.google.gson.JsonObject;
+import com.google.gson.*;
```
Comment (alfonsomunozpomer) [inline sc/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentController.java]: Try not to use wildcards in import statements. This is probably IDEA’s fault, go to *Preferences* > *Editor* > *Code Style* > *Java* and set a ridiculous high number to replace import statements with asterisks.

image

File: sc/src/main/javascript/bundles/experiment-page/src/SupplementaryInformationPageRoute.js (hunk 1 above)
Comment (monicajianu) [reply sc/src/main/javascript/bundles/experiment-page/src/SupplementaryInformationPageRoute.js]: Removed in the next branch :)

File: sc/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentController.java (hunk 3 above)
Comment (monicajianu) [reply sc/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentController.java]: Yes, it was IntelliJ doing it. I tried to fight it manually but I lost. Will change the settings.

**Pull request merged**